addons/web/controllers/main.py

```diff
@@ -2067,27 +2067,30 @@ def report_routes(self, reportname, docids=None, converter=None, **data):
     # Misc. route utils
     #------------------------------------------------------
     @http.route(['/report/barcode', '/report/barcode/<type>/<path:value>'], type='http', auth="public")
-    def report_barcode(self, type, value, width=600, height=100, humanreadable=0, quiet=1, mask=None):
+    def report_barcode(self, type, value, **kwargs):
         """Contoller able to render barcode images thanks to reportlab.
         Samples:
             <img t-att-src="'/report/barcode/QR/%s' % o.name"/>
             <img t-att-src="'/report/barcode/?type=%s&amp;value=%s&amp;width=%s&amp;height=%s' %
                 ('QR', o.name, 200, 200)"/>
-
+        Accepted keyword parameters:
         :param type: Accepted types: 'Codabar', 'Code11', 'Code128', 'EAN13', 'EAN8', 'Extended39',
         'Extended93', 'FIM', 'I2of5', 'MSI', 'POSTNET', 'QR', 'Standard39', 'Standard93',
         'UPCA', 'USPS_4State'
+        :param width: Pixel width of the barcode
+        :param height: Pixel height of the barcode
         :param humanreadable: Accepted values: 0 (default) or 1. 1 will insert the readable value
         at the bottom of the output image
         :param quiet: Accepted values: 0 (default) or 1. 1 will display white
         margins on left and right.
         :param mask: The mask code to be used when rendering this QR-code.
-                     Masks allow adding elements on top of the generated image,
-                     such as the Swiss cross in the center of QR-bill codes.
+        Masks allow adding elements on top of the generated image, such as the Swiss cross in
+        the center of QR-bill codes.
```

Comment:
Why rewrite this? the content doesn't seem to have changed.

Reply:
Oook, reverted to the old indentation (though inconsistent with the humanreadable parameter description).